src/UI/docu/ux-guide-repository-objects-properties-and-actions.md

```diff
@@ -0,0 +1,364 @@
+location to look for. When properties and actions are clearly grouped and
+segmented, the user learns where to find the information that is currently of
+interest. For example, all community interactions like star ratings and comments
+could be in the bottom right corner of an item, and all organizational
+meta-data, such as file size and version number, in the bottom left.
+
+Before we dove deeper into organizing the properties shown on various ILIAS
+objects, we investigated what different formats, functionalities, and use cases
+for properties exist. The table on the feature request [Streamline Object
+Properties](https://docu.ilias.de/goto_docu_wiki_wpage_7399_1357.html) was
+tremendously helpful to explore the many different types of elements.
+
+We propose to offer multiple consistent locations on UI components to make
+decisions about splitting up properties easier. Instead of one property section,
+an item could have multiple distinct ones. Possible semantic groups could be:
+
+* community interactions (star rating, comment number)
+* content information (next appointment date, available number of seats)
+* meta-data (creation date, file size, version number)
+
+Distinct groups will help users to quickly find the information relevant to them
+in multi-purpose views.
+
+If screens are built around a specific user intent, the most important property
+could be pulled to a featured position. Different UI components might have
+different kinds of featured positions. The already existing Leading Text of the
+UI Item and Important Fields of the UI Presentation Table are good examples for
+this.
+
+In places where we know the user intent to some extent, some (or all) irrelevant
+properties could be hidden. For example, the number of recent posts in a forum
+object is a relevant representation of activity, while the number of recent
+items in a category is usually not of interest.
+
+In some instances we can avoid repeating redundant information. For example,
+when the repository shows all categories grouped together, it might suffice to
+put the the category icon only once into the group panel headline instead of on
+every single object.
+
+We have yet to explore if and how we want to incorporate other elements of the
+interface to help display or hide information of an object outside of the item
+itself. For example, many file managers can display additional information and
+actions in a sidebar. Maybe the slate can be filled with new tools and features
+to aid in analyzing, comparing, and managing repository items.
+
+We also might want to more clearly and consistently decide on how a property is
+displayed - e.g. as a key text & value text, key text & icon, icon & value text,
+just the value text, or just as an icon. For example in "Status: Offline", the
+word "Status" seems to be redundant, as the word "Offline" alone communicates
+the same amount of information without the leading key.
+
+### User adjusted views
```

Comment:
Consider phrasing this "Personalisation"

Reply:
Changed accordingly. Thanks!